charts/spire/charts/spiffe-oidc-discovery-provider/templates/_helpers.tpl

```diff
@@ -91,3 +91,21 @@ Create the name of the service account to use
 {{- define "spiffe-oidc-discovery-provider.workload-api-socket-path" -}}
 {{- printf "/spiffe-workload-api/%s" .Values.agentSocketName }}
 {{- end }}
+
+{{- define "spiffe-oidc-discovery-provider.tls-count" -}}
```

Comment:
Is it possible to structure this so this check to runs once pre deploy? And all the templates that need to know if tls is enabled can have a simpler template `is-tls` thats just an or of the enabled flags.

Reply:
Unfortunately not.

Reply:
Well..... Maybe....Let me take a stab at that...

Reply:
Updated.